.github/workflows/docs.yaml

```diff
@@ -0,0 +1,61 @@
+name: Documentation
+# FOLLOWING https://coderefinery.github.io/documentation/gh_workflow/
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [closed] # Run conditionals based on opened/merged PRs
+    paths:
+      - 'docs/*'  # Trigger the workflow only when changes are made in the 'docs' directory
```

Comment:
this actually didn't trigger a run with two ** and it's not clear to me why since ** is recursive and should be more inclusive https://gist.github.com/bennlee/92ffaa27eea326da46e538a80c3eebae

Reply:
does it work in the below case; lines 12--16?